README.md

```diff
@@ -430,15 +389,113 @@ When **running the CLI from your CI pipeline** you will need to export two varia
 Go to your Settings page in Checkly and grab a fresh API key from [the API keys tab](https://app.checklyhq.com/settings/user/api-keys) and your
 Account ID from the [Account settings tab](https://app.checklyhq.com/settings/account/general).
 
-# Constructs API
+# Using the Constructs API
+
+All resources you can create and manage using the Checkly CLI are derived from "constructs". These constructs are just 
+[TypeScript classes](https://github.com/checkly/checkly-cli/tree/main/package/src/constructs) like `ApiCheck` in `api-check.ts` and 
```

Comment:
I'm not sure if users will actually want to look at the code to get started, and I'm not sure if it's worth mentioning the file names.

What about something like:

> To manage resources in Checkly from the Checkly CLI you can use "constructs". These constructs are JS/TS classes that represent resources like checks, alert channels, and check groups in Checkly.  You can use standard JS/TS programming when creating constructs: loops, variables, if-statements, file imports, extensions etc. When running `npx checkly deploy`, your monitoring setup in Checkly will be updated to match the constructs defined in your code.

Reply:
Assuming that we keep the README up to date, I think that most users won't need to look at the code.